src/validation/__tests__/NoDeprecatedCustomRule-test.js

```diff
@@ -8,112 +8,127 @@ import { NoDeprecatedCustomRule } from '../rules/custom/NoDeprecatedCustomRule';
 
 import { expectValidationErrorsWithSchema } from './harness';
 
-function expectErrors(queryStr) {
-  return expectValidationErrorsWithSchema(
-    schema,
-    NoDeprecatedCustomRule,
-    queryStr,
-  );
-}
-
-function expectValid(queryStr) {
-  expectErrors(queryStr).to.deep.equal([]);
-}
+function buildAssertion(sdlStr: string) {
+  const schema = buildSchema(sdlStr);
+  return { expectErrors, expectValid };
 
-const schema = buildSchema(`
-  enum EnumType {
-    NORMAL_VALUE
-    DEPRECATED_VALUE @deprecated(reason: "Some enum reason.")
-    DEPRECATED_VALUE_WITH_NO_REASON @deprecated
```

Comment:
Setting up default `deprecationReason` is tested in `buildASTSchema-test.js` so no need to duplicate it here.